com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/message/Messages.java

```diff
@@ -332,6 +332,9 @@
     public static final String SERVICE_BINDINGS_EXISTS = "Service bindings \"{0}\" exists";
     public static final String PREPARING_APPS_DEPLOYMENT = "Preparing applications deployment...";
     public static final String APPS_DEPLOYMENT_PREPARED = "Applications deployment prepared";
+    public static final String COMPUTING_NEXT_APPS_FOR_PARALLEL_DEPLOYMENT = "Computing applications for next parallel deployment...";
+    public static final String APPS_FOR_NEXT_PARALLEL_ITERATION = "Applications for next parallel iteration: {0}";

```

Comment:
Merge this message into the one below and use a consistent term: either "deployment" or "iteration"